python/llm/example/llama_simple.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This would makes sure Python is aware there is more than one sub-package within bigdl,
+# physically located elsewhere.
+# Otherwise there would be module not found error in non-pip's setting as Python would
+# only search the first bigdl package and end up finding only one sub-package.
+
+from bigdl.llm.ggml.model.llama import Llama
```

Comment:
Change to
```python
from bigdl.llm.models import LLaMA
```

Reply:
- This new API is not ready yet. Once its ready I'll change it in langchain integrations instead.
- These simple examples (e.g. llama_simple and redpajama_simple ) are just for validation purposes (langchain users won't use these llm APIs directly). I plan to make them into UTs once CI/CD is ready.

@TheaperDeng please inform people who's working on examples to also use the new interface bigdl.llm.models.

Reply:
@plusbang is working on the namespace change (`from bigdl.llm.models import LLaMA`).
CICD has now been enabled on Linux, we will work on windows CICD next week.